docs/Course Wiki/PHYS Course Offerings/PHYS213.md

```diff
@@ -0,0 +1,56 @@
+# PHYS213
+
+PHYS213 (Thermal Physics) is a 2-credit hour, 8-week long, course that is required for physics majors. It is required for ECE majors. It is offered in the fall, spring, and summer semesters.
```

Comment:
change to "(University Physics: Thermal Physics)", also change to "that is required for ECE majors as a part of the Foundational Mathematics and Science requirement.  It is also required for physics majors."  Also mention it is a second 8-weeks course.